hooks/__test__/useCopyToClipboard.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import { FormattedMessage } from 'react-intl';
+import { useCopyToClipboard } from '../useCopyToClipboard';
+import { LocaleProvider } from '../../providers/localeProvider';
+import { AppProps } from '../../types';
+
+const mockWriteText = jest.fn();
+const i18nData = { currentLocale: { code: 'en' } } as AppProps['i18nData'];
+
+Object.defineProperty(window, 'navigator', {
```

Comment:
Same here from my previous comments

Reply:
Please, this still needs to be fixed. It would be best if you did the same as from here: https://github.com/nodejs/nodejs.org/pull/5234/files#diff-71105d987a7c5eebb71d80e5a58ec56567f571a11a7f13aee6ef3d4aecfc0895R8